### ISSUE TITLE
PlaidLink rendering "Error: Element type is invalid: expected a string"

### ISSUE THREAD

**Issue by azzayan** [plaid/react-native-plaid-link-sdk#337]
# The problem

Hello! I've followed all the instructions for upgrading to version 7, but when I use `<PlaidLink>` on the client side, it gives me this error. This happens after I've already modified my server to use `link_token` and I try to pass `link_token` through the `tokenConfig` prop.

**Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in.**

I should note that I'm currently using [React Native Plaid Link](https://www.npmjs.com/package/react-native-plaid-link), and they have their own component called `PlaidAuthenticator`. But with this new upgrade and the fact that React Native Plaid Link hasn't yet accounted for it, I figured I should just switch to the real thing.

Also, I'm using regular JS files as opposed to TypeScript.

This error happens on Android and iOS.

# Environment

|                        |  |
| ------------------------| ---------- |
| Plaid Link React Native          | 7.0.0                |
| ReactNative Version     | 0.62                |
| Occurs on Android             | yes                |
| Android OS Version                    | 10                |
| Android Devices/Emulators    | Pixel 2 emulator                |
| Occurs on iOS     | yes                |
| iOS Version           | 13.2                |
| iOS Devices/Emulators               | iPhone 11                |

# Screenshots

![Simulator Screen Shot - iPhone 11 - 2021-03-04 at 14 28 49](https://user-images.githubusercontent.com/16055193/110026625-7b750e00-7cf6-11eb-9dda-167ef69afa88.png)

# Code To Reproduce Issue

Here's the relevant code in my React component:
```
import {PlaidLink} from 'react-native-plaid-link-sdk';

...

<PlaidLink
  tokenConfig={{
    token: myPlaidLinkToken,
  }}
  onSuccess={() => {
    console.log('success');
  }}
  onExit={() => {
    console.log('exit');
}}>
```

**Comment by Symyon** [comment]
I have the same thing, trying to migrate from v5 to v7.

**Comment by Symyon** [comment]
I tried the example provided with this lib, and it has the same issue. Is there anything 'wrong' in our environment? 

ReactNative Version | 0.62
-- | --

iOS Version | 11
-- | --

**Comment by Symyon** [comment]
Same here, I'm using 13.5 not 11. But same thing cones on Android, so I don't think it's iOS related.

**Comment by vaughansharman** [comment]
Also experiencing this issue. Has anyone found a fix yet?

**Comment by vaughansharman** [comment]
I have a workaround for the moment. Just don't use PlaidLink

```
export const MyPlaidComponent = (props: MyPlaidComponentProps) => {

  const plaidProps: PlaidLinkProps = {
        tokenConfig:{
            token: props.token,
        },
        onSuccess: (success: LinkSuccess) => { console.log(success) },
        onExit: (exit: LinkExit) => { console.log(exit) }
    }
    openLink(plaidProps)
    return(<View>
            <Text>Foobar</Text>
        </View>)
};
```

**Comment by azzayan** [comment]
@vaughansharman, thanks for the response. It doesn't seem to be working for me, though. The response from `openLink()` isn't coming back.

```
import {openLink} from 'react-native-plaid-link-sdk';

...

const openPlaidLink = () => {
  console.log('open plaid link');  // this logs

  setTimeout(
    () =>
      openLink({
        clientName: 'my client name',
        env: 'sandbox',
        onSuccess: () => {
          console.log('plaid success');
        },
        product: 'auth',
        publicKey: PLAID_PUBLIC_KEY,
      }),
    100,
  );

  console.log('after openLink'); // this doesn't log
};

...

<View>
  <Button
    title={'Add Account'}
    onPress={() => openPlaidLink}
  />
</View>
```

**Comment by Symyon** [comment]
Thx @vaughansharman ! Works.

**Comment by Symyon** [comment]
Looking at RN versions changelog, Pressable component was added in RN 0.63 and in Plaid migration steps was mentioned that this is used: "PlaidLink renders a Pressable component, which wraps the component you provide and intercepts onPress events to open Link." So RN 0.63 is min requirement? Maybe that is the issue. Weird though that the example is using RN 0.63 but throws the same error.

**Comment by azzayan** [comment]
As @Symyon mentioned, the latest version of this library uses a Pressable component, and Pressable was [introduced](https://programmingwithmosh.com/react-native/learn-the-new-pressable-component-in-react-native/) in React Native 0.63. I would guess that that's the issue, so thanks, @Symyon.

I had resisted upgrading React Native because it was creating more pain for me than I expected, but this [excellent guide](https://react-native-community.github.io/upgrade-helper/) really helped me through it. I was able to upgrade to React Native 0.64, and now I'm using PlaidLink.

Thanks to everyone for the help! I want to confirm that you indeed must have at least React Native 0.63 to use PlaidLink. If so, then we can close this issue.
